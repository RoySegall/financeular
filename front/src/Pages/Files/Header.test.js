describe('File header component', () => {

  it('Testing the correct month is displayed', () => {
    expect(1).toBe(2);
  })

});