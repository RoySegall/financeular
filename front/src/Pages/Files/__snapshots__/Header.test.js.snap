// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File header component Testing the correct month is displayed 1`] = `
<div
  className="flex justify-between items-center"
>
  <h1
    className="text-4xl pt-4 text-left font-rubik"
  >
    Results for the 
    January 2019
  </h1>
  <div>
    Viewing mode: 
    <u>
      Balance
    </u>
    . Switch to view 
    <u>
      Insights
    </u>
  </div>
</div>
`;
